datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -262,13 +262,38 @@ func (f *ChainDataFetcher) stopRangeFetching() error {
 	return nil
 }
 
+func (f *ChainDataFetcher) retryMakeChainEvent(blockNumber uint64) (blockchain.ChainEvent, error) {
+	var (
+		retryMax      = 100
+		retryInterval = 1 * time.Second
+	)
+	ev, err := f.makeChainEvent(blockNumber)
+	for i := 1; err != nil; i++ {
+		select {
+		case <-f.stopCh:
+			return ev, err
+		default:
+			logger.Warn("retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", i, "retryMax", retryMax, "err", err)
```

Comment:
I wonder why you named it "i", not "retryCount".

Reply:
I think `retryCount` is better to read. Reflected, thanks :)